packages/office-ui-fabric-react/src/components/Spinner/Spinner.types.ts

```diff
@@ -100,6 +112,7 @@ export interface ISpinnerStyleProps {
   theme: ITheme;
   size?: SpinnerSize;
   className?: string;
+  labelPosition?: SpinnerLabelPosition;
```

Comment:
It would be great to document this.

Reply:
(p.s. really all of these should be documented but Spinner is one of the components that are a bad example :(...)